includes/Integrations/Jetpack.php

```diff
@@ -131,7 +131,7 @@ public function filter_ajax_query_attachments_args( array $args ) {
 		$allowed_mime_types = $this->get_allowed_mime_types();
 		$allowed_mime_types = array_merge( ...array_values( $allowed_mime_types ) );
 		if ( in_array( self::VIDEOPRESS_MIME_TYPE, $args['post_mime_type'], true ) && ! array_diff( $allowed_mime_types, $args['post_mime_type'] ) ) {
-			add_filter( 'wp_prepare_attachment_for_js', [ $this, 'filter_admin_ajax_response' ], 10, 2 );
+			add_filter( 'wp_prepare_attachment_for_js', [ $this, 'filter_admin_ajax_response' ], 15, 2 );
```

Comment:
Load after https://github.com/google/web-stories-wp/blob/ff201b1be19e5903a31bea40a3b0bfd9a83e15e1/includes/Media/Media.php#L230

Reply:
Could you add this explanation as an inline comment perhaps? So we don't accidentally change it again.

Reply:
Added comment https://github.com/google/web-stories-wp/pull/7691/commits/b6640130f238ca998caf061cbfc53c2091a97650